src/client/near.rs

```diff
@@ -186,6 +186,26 @@ impl NearClient {
         .await
     }
 
+    pub async fn contract_call_with_deposit(
```

Comment:
`fn contract_call` could be implemented with `contract_call_with_deposit` with `deposit = 0`.

Reply:
Fair enough.